### PR TITLE
Bee

### PR DESCRIPTION
**Pull request by CoderBlee**

### CODE REVIEW

**Comment by CoderBlee** [comment]
Changes are done and updated. You do not have a contribution.md  file and the issues template is joined with .githud when you open with vs code, kindly look into it as it is not the same when you view from GitHub

File: .github/ISSUE_TEMPLATE/task.md
Comment (kjaymiller) [inline .github/ISSUE_TEMPLATE/task.md:1]: The changes to the ISSUE Template are filled in for a particular issue and not with placeholders. This should be cleared out or templates should be filled.

File: how_to/contribute_to_the_bpd_community.md [hunk 1]
```diff
@@ -1 +1,92 @@
 # How to Contribute to the BPD Community
+Contributing to open source can be a rewarding experience, and getting started is easier than you might think. Here’s a beginner-friendly guide to help you navigate your way into the open source world:
+
+### 1. **Understand What Open Source Is**
+
+Open source software is developed collaboratively and made available to anyone to use, modify, and distribute. Contributions can range from fixing bugs and adding features to improving documentation.
+
+### 2. **Find a Project**
+
+#### **Explore Repositories**
```
Comment (kjaymiller) [inline how_to/contribute_to_the_bpd_community.md:10]: This Section talks about contribute to open source and not particularly the BPD projects which are all on GitHub. This should be made more specific to contributing to Black Python Devs.

You can support that with links that help contributors file issues or prs

File: how_to/contribute_to_the_bpd_community.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+
+- **GitHub**: Popular platform where many open source projects are hosted. Explore repositories through the [Explore page](https://github.com/explore) or by searching for topics that interest you.
+- **GitLab**: Another platform similar to GitHub. Look for projects via [GitLab Explore](https://gitlab.com/explore/projects).
+- **Bitbucket**: Also hosts open source projects. Check out repositories via [Bitbucket’s Explore page](https://bitbucket.org/product).
+
+#### **Look for Good First Issues**
+
+Many projects label issues that are suitable for beginners. Search for labels like “good first issue,” “beginner,” or “help wanted.”
```
Comment (kjaymiller) [inline how_to/contribute_to_the_bpd_community.md:18]: [nit] you can also add some pre-configured search links for repos in our org with label "good first issue"

File: how_to/contribute_to_the_bpd_community.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+
+### 3. **Understand the Project**
+
+#### **Read the Documentation**
+
+- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the read me first: https://github.com/CoderBlee/blackpythondevs
```
Comment (kjaymiller) [inline how_to/contribute_to_the_bpd_community.md:24]:
```suggestion
- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the read me first: https://github.com/blackpythondevs/blackpythondevs
```

File: how_to/contribute_to_the_bpd_community.md [hunk 4]
```diff
@@ -1 +1,92 @@
...
+- **CONTRIBUTING.md**: Guidelines for contributing to the project. This can include coding standards, testing protocols, and how to submit changes.
+- **CODE_OF_CONDUCT.md**: Code of conduct outlines the expectations for behavior in the community.
+
+#### **Set Up Your Development Environment**
+
+Follow the setup instructions provided in the documentation. This may involve installing dependencies, setting up a development environment, or running tests.
+
+### 4. **Start Contributing**
+
+#### **Fork and Clone the Repository**
```
Comment (kjaymiller) [inline how_to/contribute_to_the_bpd_community.md:34]: This Section should likely come before the setup your environment

File: how_to/join_leadership_council.md
Comment (kjaymiller) [inline how_to/join_leadership_council.md:1]: This was updated by the documentation team in another PR. It should be removed from this one.

File: policies/paying-for-things-and-getting-reimbursed.md [hunk 5]
```diff
@@ -34,6 +34,7 @@ All wire transfers require the following information:
 * address of their bank
 * SWIFT code of their bank (if applicable)
 * currency of their account
+* paypal payment, voucher
```
Comment (kjaymiller) [inline policies/paying-for-things-and-getting-reimbursed.md:37]:
```suggestion

```

Comment (kjaymiller) [reply policies/paying-for-things-and-getting-reimbursed.md:37]: That would be for paypal requests only. There should be an issue that adds instructions for PayPal and WISE (the other two repayment methods we support)

**Review by kjaymiller — CHANGES_REQUESTED** [review]
I have left a few comments that all need to be addressed. This should not be merged before these changes are made.

**Comment by kjaymiller** [comment]
@CoderBlee - Just a reminder and perhaps something that should be mentioned to your open source cohort.

It's best to make small PRs that address a single issue (or 2 if they are connected) this will prevent lots of work being required to fix issues.

File: policies/paying-for-things-and-getting-reimbursed.md (hunk 5 above)
Comment (CoderBlee) [reply policies/paying-for-things-and-getting-reimbursed.md:37]: Okay, I will look into it also how do I make an issue?  I have never done it before

File: how_to/contribute_to_the_bpd_community.md (hunk 1 above)
Comment (CoderBlee) [reply how_to/contribute_to_the_bpd_community.md:10]: Okay, I am going to implement the changes. Kindly give me time. Everything will be fixed soon.

File: policies/paying-for-things-and-getting-reimbursed.md (hunk 5 above)
Comment (kjaymiller) [reply policies/paying-for-things-and-getting-reimbursed.md:37]: https://github.com/BlackPythonDevs/blackpythondevs/issues/new

Add information like:
- What is the current issue
- (If Possible) how can this issue be resolved

Comment (CoderBlee) [reply policies/paying-for-things-and-getting-reimbursed.md:37]: Sorry for my late respond I was not feeling well and had exams. I have created the issue and the changes needed, I saw that they were made already , should I still do them or leave them

File: how_to/contribute_to_the_bpd_community.md (hunk 4 above)
Comment (CoderBlee) [reply how_to/contribute_to_the_bpd_community.md:34]: I was not well and have exams, I saw that someone already have made the changes. Should I still do the changes you have requested or look into other issues

Comment (kjaymiller) [reply how_to/contribute_to_the_bpd_community.md:34]: I haven't seen if all the changes have been made. I would keep your code and address the issues with individual PRs for anything still active.

Comment (CoderBlee) [reply how_to/contribute_to_the_bpd_community.md:34]: Can I still just implement what you said and make the changes regardless?

**Comment by kjaymiller** [comment]
As mentioned before please limit your PRs to solving singular issues as it makes it easier for maintainers to return.

I'm closing this issue. Please do not reopen this issue.

**Pull request closed without merging**